lms/djangoapps/courseware/views/views.py

```diff
@@ -1176,15 +1181,12 @@ def _certificate_message(student, course, enrollment_mode):  # lint-amnesty, pyl
     if cert_downloadable_status['is_generating']:
         return GENERATING_CERT_DATA
 
-    if cert_downloadable_status['is_unverified']:
-        return UNVERIFIED_CERT_DATA
+    if cert_downloadable_status['is_unverified'] or _missing_required_verification(student, enrollment_mode):
+        return _unverified_cert_data(cert_downloadable_status)
```

Comment:
We're getting close - The argument also needs to be removed when the method is called. I believe this is why the unit tests are failing.

Reply:
Re-corrected